### PR TITLE
Add flutter example

### PR DESCRIPTION
**Pull request by smkhalsa**
Separating this out per https://github.com/gql-dart/gql/pull/32#issuecomment-575930496.

I've removed the fragment-related code and created a [model](https://github.com/smkhalsa/gql/blob/flutter-example/gql_example_flutter/lib/src/pokemon_card/pokemon.dart) manually that accepts a Json object. This is probably what I would do if I didn't have built fragments.

### CODE REVIEW

File: gql_example_flutter/test/widget_test.dart [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// This is a basic Flutter widget test.
```
Comment (klavs) [inline gql_example_flutter/test/widget_test.dart]: This file should probably be dropped.

File: gql_example_flutter/pubspec.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
+name: gql_example_flutter
+description: A new Flutter project.
```
Comment (klavs) [inline gql_example_flutter/pubspec.yaml]: You can drop the `description` and run `dart ./multipack/bin/main.dart pubspec clean` before commit. It adds some stupid EOL whitespaces my editor tends to delete, but at least it guarantees consistency and reduces git diff noise.

File: gql_example_flutter/pubspec.yaml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  # The following adds the Cupertino Icons font to your application.
+  # Use with the CupertinoIcons class for iOS style icons.
+  cupertino_icons: ^0.1.2
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  build_runner: ^1.0.0
+  gql_pedantic: ^1.0.0
+  gql_build:
```
Comment (klavs) [inline gql_example_flutter/pubspec.yaml]: Please use the published version.

For local development, run `dart ./multipack/bin/main.dart pubspec override` and add the missing dependency override.

File: gql_example_flutter/README.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# gql_example_flutter
+
+A new Flutter project.
```
Comment (klavs) [inline gql_example_flutter/README.md]: Could you update this to briefly show how to build `gql`-specific parts?

It's ok to expect that the readers know what Flutter is and how to run a Flutter app (in simulator/emulator or on the device).

**Comment by klavs** [comment]
Oh, and please add this to `.github/labeler.yml` config

File: gql_example_flutter/test/widget_test.dart (hunk 1 above)
Comment (smkhalsa) [reply gql_example_flutter/test/widget_test.dart]: complete in https://github.com/gql-dart/gql/pull/33/commits/753e3957e9a633af214c219a61918289341ce901

File: gql_example_flutter/pubspec.yaml (hunk 2 above)
Comment (smkhalsa) [reply gql_example_flutter/pubspec.yaml]: done in 719a806

File: gql_example_flutter/pubspec.yaml (hunk 3 above)
Comment (smkhalsa) [reply gql_example_flutter/pubspec.yaml]: done in 762961b

File: gql_example_flutter/README.md (hunk 4 above)
Comment (smkhalsa) [reply gql_example_flutter/README.md]: done in https://github.com/gql-dart/gql/pull/33/commits/fb3e77638090b3737222e9361f76987b3fedf946

**Comment by smkhalsa** [comment]
> Oh, and please add this to `.github/labeler.yml` config

complete in https://github.com/gql-dart/gql/pull/33/commits/faf46a173d9459f2e536ec6075b47190b53a4d3c

**Pull request merged**